src/main/java/frc/robot/util/CompoundCommands.java

```diff
@@ -277,15 +299,15 @@ public static DeferredCommand deferDrive(Supplier<Command> command) {
   public static Command armToGoal(double wristSetPos, double pivotSetPos, double elevatorSetPos) {
```

Comment:
I think you can remove the detailed description of the motion in the comments above, the code is quite readable.

Reply:
Postponing to https://github.com/AEMBOT/FRC_2025/issues/61